### PR TITLE
Upgraded to storybook 7.3.2

### PR DESCRIPTION
**Pull request by juanparati**
I upgrade to storybook 7.3.2. I also fix the issue with SSL validation.

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
   stories: [
     "../stories/**/*.stories.mdx",
     "../stories/**/*.stories.@(js|jsx|ts|tsx)",
   ],
+
   addons: [
-    "@storybook/addon-links",
-    "@storybook/addon-essentials",
-    "@storybook/addon-interactions",
+    getAbsolutePath("@storybook/addon-links"),
```
Comment (zyc9012) [inline .storybook/main.js:9]: I removed all the `getAbsolutePath` wraps and it still worked. Why do we need it?

File: babel.config.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      "@babel/preset-env",
+      {
+        "targets": {
+          "chrome": 100,
+          "safari": 15,
+          "firefox": 91
+        }
+      }
+    ],
+	"@babel/preset-flow",
```
Comment (zyc9012) [inline babel.config.json:13]: Let's format this file

File: packages/core/src/modules/cell.ts [hunk 3]
```diff
@@ -153,10 +153,6 @@ export function setCellValue(
         delete cell.f;
       }
 
-      // if (!_.isNil(v.spl)) {
```
Comment (zyc9012) [inline packages/core/src/modules/cell.ts:156]: Let's not edit files unrelated to this PR

**Review by zyc9012 — CHANGES_REQUESTED** [review]
Thanks @juanparati for upgrading storybook 👍.

Let's also fix type deprecations in story files, something like this

```diff
diff --git a/stories/API.stories.tsx b/stories/API.stories.tsx
index a54c1aa..4a1a3b9 100644
--- a/stories/API.stories.tsx
+++ b/stories/API.stories.tsx
@@ -1,11 +1,11 @@
 import React, { useCallback, useRef, useState } from "react";
-import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Meta, StoryFn } from "@storybook/react";
 import { Workbook, WorkbookInstance } from "@fortune-sheet/react";
 import { Sheet } from "@fortune-sheet/core";
 
 export default {
   component: Workbook,
-} as ComponentMeta<typeof Workbook>;
+} as Meta<typeof Workbook>;
 
 const ApiExecContainer: React.FC<{
   onRun: () => any;
@@ -43,7 +43,7 @@ const ApiExecContainer: React.FC<{
   );
 };
 
-export const GetCellValue: ComponentStory<typeof Workbook> = () => {
+export const GetCellValue: StoryFn<typeof Workbook> = () => {
   const ref = useRef<WorkbookInstance>(null);
   const [data, setData] = useState<Sheet[]>([
     {
```

----

The default pages becomes "Docs", which is weird, could you fix that?

image

**Comment by juanparati** [comment]
@zyc9012: All changes are done:
- Reverted unrelated [28bacdb](https://github.com/ruilisi/fortune-sheet/pull/470/commits/28bacdb41ce18e9c2a28f5385ce9090ea002cdc7) and [b775c6d](https://github.com/ruilisi/fortune-sheet/pull/470/commits/b775c6dd4b209263d51d25754bf9a27b90cbad44)
- Replaced deprecated types
- Removed Docs link from storybook
- Removed "getAbsolutePath" method
- Reformatted babel.config.json

File: packages/core/src/modules/cell.ts [hunk 4]
```diff
@@ -143,6 +143,10 @@ export function setCellValue(
 
   let vupdate;
 
+  if (!_.isNil(v.v) && _.includes(v.ct?.fa, "0.0")) {
```
Comment (zyc9012) [inline packages/core/src/modules/cell.ts:146]: Let's not introduce unrelated code here.

**Review by zyc9012 — CHANGES_REQUESTED** [review]
Thanks @juanparati 

Let's also update `Collabration.stories.tsx` and `Features.stories.tsx`

**Comment by juanparati** [comment]
@zyc9012 : Changes made:
- Removed unrelated
- Replaced deprecated types for stories
- Replaced autodocs to "tag"

**Review by zyc9012 — APPROVED** [review]
Thanks @juanparati 👍

**Pull request merged**